lisa/schema.py

```diff
@@ -580,6 +580,51 @@ def _generate_min_capability(self, capability: Any) -> Any:
         return min_value
 
 
+@dataclass_json()
+@dataclass()
+class NvmeOptionSettings(FeatureSettings):
```

Comment:
It's not a feature like disk or network, which are referenced by node space. So it should be moved into its feature file, `lisa/features/nvme.py`.

Reply:
ok

Reply:
moved